### PR TITLE
fix(notice):add functional onClose notice story

### PR DESCRIPTION
**Pull request by devaneereid**
## Jira issue
[FE: New UI changes to the login pages](https://myzonos.atlassian.net/jira/software/projects/CLSFY/boards/65?assignee=603058259f84a900690cdd2b&selectedIssue=CLSFY-1780)

## Description
This PR adds a functioning onClose to the Notify story in amino. 

## Todo

- [x] Bump version and add tag

### CODE REVIEW

**Review by goodpickle — CHANGES_REQUESTED** [review]
The toggle notice doesn't toggle the notice, it only opens the notice. I'm Okay if you change the button label or have it actually toggle the notice open or closed.

File: src/stories/Notice.stories.tsx [hunk 1]
```diff
@@ -12,21 +13,29 @@ const NoticeStories: Meta = {
...
-  onClose,
-}: NoticeProps) => {
+const Template: Story<NoticeProps> = ({ intent, children }: NoticeProps) => {
+  const [open, setOpen] = useState(true);
   return (
-    <Notice intent={intent} onClose={onClose}>
-      {children}
-    </Notice>
+    <>
+      <Button onClick={() => (!open ? setOpen(true) : setOpen(false))}>
```
Comment (goodpickle) [inline src/stories/Notice.stories.tsx:20]: What do you think about simplifying this logic?
```suggestion
      <Button onClick={() => setOpen(!open)}>
```

**Review by goodpickle — CHANGES_REQUESTED** [review]

File: src/stories/Notice.stories.tsx (hunk 1 above)
Comment (devaneereid) [reply src/stories/Notice.stories.tsx:20]: That's much cleaner, thank you!

**Comment by goodpickle** [comment]
It looks like the prettier linter is preventing a build. Go ahead and fix that and then do a `yarn version:patch` and we will get this out.

**Review by goodpickle — DISMISSED** [review]

**Comment by goodpickle** [comment]
@devaneereid It looks like there are some merge conflicts that need to be resolved.

**Review by Thienhuynh95 — APPROVED** [review]

**Pull request merged**